src/helpers/daily-data-providers/combined-data-collection-settings.ts

```diff
@@ -0,0 +1,21 @@
+import MyDataHelps, {
+    DataCollectionSettings as MDHDataCollectionSettings,
+    SupportedDeviceDataV2DataType,
+} from "@careevolution/mydatahelps-js";
+
+export interface CombinedDataCollectionSettings {
+    settings: MDHDataCollectionSettings;
+    deviceDataV2Types: SupportedDeviceDataV2DataType[];
+}
+
+export async function getCombinedDataCollectionSettings(): Promise<CombinedDataCollectionSettings> {
+    const [settings, deviceDataV2Types] = await Promise.all([
+        MyDataHelps.getDataCollectionSettings(),
+        MyDataHelps.getDeviceDataV2AllDataTypes(),
+    ]);
+
+    return {
+        settings,
```

Comment:
Rather than having a settings object with a settings property.  Maybe we use `...settings` here and adjust the interface definition above to be:
```
export interface CombinedDataCollectionSettings extends DataCollectionSettings {
    deviceDataV2Types: SupportedDeviceDataV2DataType[];
}
```

Reply:
I do like that a bit better as well now that I think about it (and maybe that is what we should just do server-side as well)